### ISSUE TITLE
Create Asciidoclet sample project

### ISSUE THREAD

**Issue by msgilligan** [asciidoctor/asciidoclet#78]
https://github.com/JGrenier/asciidoclet-sample is nice, but we should have a similar project under the asciidoctor organization that we can keep up to date and integrate with CI tools for testing and demonstration purposes.

**Comment by msgilligan** [comment]
Maybe we could just start using Asciidoclet in all the existing examples. 

I'd like to see a using-everything-asciidoctor project that kind-of mirrors the real-world usage I've been using on some of my projects (e.g. [ConsensusJ]( https://github.com/ConsensusJ/consensusj)) I also want an example that builds with Java 9/10 on Gitlab CI -- we can't do that for Asciidoclet itself, but we can do it for projects that use the latest Asciidoclet.

**Comment by johncarl81** [comment]
We've tried to make the Asciidoclet project itself a self-example of using Asciidoclet... do you think a separate project would be able to highlight more or a different angle?

**Comment by msgilligan** [comment]
Yeah, the "dog food" approach is a good idea and we should definitely continue to do that.

>  I also want an example that builds with Java 9/10 on Gitlab CI -- we can't do that for Asciidoclet itself, but we can do it for projects that use the latest Asciidoclet.

I suppose it might be possible to use Java 9 in Java 8 mode to build it. I'm really interested in being able to produce the new HTML5 output. I'll look into that...
